packages/react-juce/src/components/TextInput.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { PropsWithChildren } from 'react';
+
+export interface InputEvent {
+  value: string
+}
+
+export interface ChangeEvent {
+  value: string
+}
+
+export interface TextInputProps {
+  value?: string,
+
+  color?: string,
+  fontSize?: number,
+  fontStyle?: number,
+  fontFamily?: string,
+  justification?: number,
+  'kerning-factor'?: number,
```

Comment:
Probably not a concern for this PR, but I think we should include some mechanism in https://github.com/nick-thompson/blueprint/blob/master/packages/react-juce/src/lib/Backend.ts for taking any relevant keys from interfaces under components/* for auto camelCase translation.  We only really do css properties at the moment.  That way props can have a consistent case.  I'll raise something for that soon.